packages/sensors_plus/sensors_plus/example/integration_test/sensors_plus_test.dart

```diff
@@ -22,4 +22,64 @@ void main() {
     });
     expect(await completer.future, isNotNull);
   });
+
+  testWidgets('Can subscribe to gyroscopeEvents and get non-null events',
+      (WidgetTester tester) async {
+    final completer = Completer<GyroscopeEvent>();
+    StreamSubscription<GyroscopeEvent> subscription;
+    subscription = gyroscopeEvents.listen((GyroscopeEvent event) {
+      completer.complete(event);
+      subscription.cancel();
+    });
+    expect(await completer.future, isNotNull);
+  });
+
+  testWidgets('Can subscribe to gyroscopeEvents and get expected events',
+      (WidgetTester tester) async {
+    final completer = Completer<GyroscopeEvent>();
+    StreamSubscription<GyroscopeEvent> subscription;
+    subscription = gyroscopeEvents.listen((GyroscopeEvent event) {
+      completer.complete(event);
+      subscription.cancel();
+    });
+    print(await completer.future);
```

Comment:
I tried to create a test that verifies an event from an emulator device, and you can see it as the code shows at line 45(event from the emulator).

At last, I got a value `[GyroscopeEvent (x: 0.0, y: 0.0, z: 0.0)]`. Then I create a `completer2` to store it as you see.

```bash
    expectSubscription = gyroscopeEvents.listen((GyroscopeEvent event) {
      completer2.complete(GyroscopeEvent(0.0, 0.0, 0.0));
      expectSubscription.cancel();
    });
```

I gave the values to the `completer2`. Then I compared it with `completer`.

```bash
    expect(await completer.future, await completer2.future);
```

And the exception logs are as follows:

<img width="871" alt="image" src="https://user-images.githubusercontent.com/43690894/187227570-c5909e38-a00c-44ee-92ec-5b2a603cbcaf.png">

This error message struck me as bizarre. (After trying hard for over two hours)